src/utils/types.ts

```diff
@@ -0,0 +1,23 @@
+import {DangerDSLType, fail} from 'danger'
+
+export type RuleParamsBase = {
```

Comment:
@quesabe please create a dir `types/` with one type definition per file. Export all types in `types/index.ts`, so that they can be imported as following:
```typescript
import RuleParamsBase from `utils/types`
```

Reply:
I remember in templates I named the typing files with CamelCase. Hence did the same way here. Is it applicable to types, or dash-case is more relevant?